index.html

```diff
@@ -13,19 +13,39 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <link rel="stylesheet" href="style.css">
     </head>
+
     <body>
+        <!--navbar fixed top-->
+        <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Awesome Books</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse mx-auto" id="navbarNavAltMarkup">
+                    <div class="navbar-nav ml-auto">
+                        <a onclick="Show('list')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="listLink" class="nav-link active" aria-current="page" href="#">List</a>
+                        <a onclick="Show('add')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="addLink" class="nav-link" href="#">Add new</a>
+                        <a onclick="Show('contact')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="contactLink" class="nav-link" href="#">Contact</a>
+                    </div>
+                </div>
+            </div>
+        </nav>
 
-        <h2>All awesome books</h2>
-        <!--content generated with js TABLE-->
-        <table class="table border" id="booksTable">
-           
-        </table>
+        <!--Datetime-->
+        <br><br><p class="text-end" id="myDate"></p>
 
-        <hr>
+        <!--List of books-->
+        <div id="listPage" class="d-block">
+            <h2>All awesome books</h2>
+            <!--content generated with js TABLE-->
+            <table class="table border" id="booksTable">
+               
+            </table>
+        </div>
 
         <!--Form to add book-->
-        <section>
-
+        <div id="addPage" class="d-none">
             <h2>Add a book</h2>
             
             <form name="saveBook" id="saveBook">
```

Comment:
- [ ] Your contact information is **not centered** on the page. Kindly ensure to center the contact information, as can be seen in the screenshot o the original below:

**Your contact page:**
![image](https://user-images.githubusercontent.com/12745474/133693128-2b0b3932-b387-406e-86ac-167895b1cb5c.png)


**The original contact page:**
![image](https://user-images.githubusercontent.com/12745474/133693085-f60948d7-3daa-4d84-9873-df01559d0d4f.png)